### ISSUE TITLE
Unable to ajax load a html file

### ISSUE THREAD

**Issue by noahpatterson** [tapio/live-server#27]
I am getting the following error (using 0.7.0 with default settings with node 0.10.36) when trying to load an html file via ajax:

```Error: Requested Range Not Satisfiable
    at SendStream.error (/Users/noahpatterson/.nvm/v0.10.36/lib/node_modules/live-server/node_modules/send/index.js:245:16)
    at SendStream.send (/Users/noahpatterson/.nvm/v0.10.36/lib/node_modules/live-server/node_modules/send/index.js:540:19)
    at onstat (/Users/noahpatterson/.nvm/v0.10.36/lib/node_modules/live-server/node_modules/send/index.js:600:10)
    at Object.oncomplete (fs.js:108:15)```

This happens in Safari, Firefox, and Chrome.

If I use a simple ruby server instead, everything loads as expected: `ruby -rwebrick -e'WEBrick::HTTPServer.new(:Port => 3000, :DocumentRoot => Dir.pwd).start'`

Here is a simple example ajaxing a html file `product.html`:

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>test</title>
</head>
<body>
  <h1>hello</h1>
  <span id="ajaxButton" style="cursor: pointer; text-decoration: underline">
    Make a request
  </span>
  <script type="text/javascript">
  (function() {
    var httpRequest;
    document.getElementById("ajaxButton").onclick = function() { makeRequest('product.html'); };

    function makeRequest(url) {
      if (window.XMLHttpRequest) { // Mozilla, Safari, ...
        httpRequest = new XMLHttpRequest();
      } else if (window.ActiveXObject) { // IE
        try {
          httpRequest = new ActiveXObject("Msxml2.XMLHTTP");
        }
        catch (e) {
          try {
            httpRequest = new ActiveXObject("Microsoft.XMLHTTP");
          }
          catch (e) {}
        }
      }

      if (!httpRequest) {
        alert('Giving up :( Cannot create an XMLHTTP instance');
        return false;
      }
      httpRequest.onreadystatechange = alertContents;
      httpRequest.open('GET', url);
      httpRequest.send();
    }

    function alertContents() {
      if (httpRequest.readyState === 4) {
        if (httpRequest.status === 200) {
          alert(httpRequest.responseText);
        } else {
          alert('There was a problem with the request.');
        }
      }
    }
  })();
  </script>
</body>
</html>
```
Thanks for your help!

**Comment by tapio** [comment]
Can you share what the `product.html` is like? Recently the injection was changed to happen just before `</body>` so perhaps it can't find that in the ajaxed file and fails. Should make sure the code falls back gracefully to serve the file uninjected instead of "crashing" when that happens.

**Comment by noahpatterson** [comment]
Yeah the file is a template file, so it only contains `<h1>Hello</h1>` there isn't a `</body>`. If there's anything I can do to help with this, let me know

**Comment by tapio** [comment]
I'm too busy this week, but if you don't hear anything next week, a reminder would be appreciated. :)

Alternatively, if you are up for fixing it yourself, [here](https://github.com/tapio/live-server/blob/1e4d94739850f7d49ade34e38eca5a1537bfa48f/index.js#L48-L55) is the code segment that I think is the cause. It should be fixed to not modify the length header (and not inject) in case `</body>` is not found.

**Comment by tapio** [comment]
I've published version 0.7.1 to npm which seems to fix this issue. Feel free to test and report any further problems. :)
